### ISSUE TITLE
As a translator, ISBAT easily translate CoBlocks strings

### ISSUE THREAD

**Issue by jrtashjian** [godaddy/coblocks#841]
As a translator, I should be able to comprehend strings within CoBlocks to easily translate the plugin into different languages.

AC:
1. Use proper placeholders if variables exist within strings:
`/* translators: %s: This is a description of what the variable is */`

2. Where necessary, strings should have proper context as explain here. This would be the case for block names, tags and anywhere context is needed.

3. Use `%d` for numerical placeholders.
